CHANGELOG.md

```diff
@@ -81,6 +82,8 @@
 
 - Stopped ignoring nanoseconds in `pd.Timedelta` scalars.
 - Fixed AssertionError in tree of binary operations.
+- Fixed a bug where `Series.reindex` and `DataFrame.reindex` did not update the result index's name correctly.
+- Fixed a bug where `Index.name` was not being set correctly during binary operations.
```

Comment:
I didn't follow this line.

Reply:
I think s different PR fixed this, so I'm removing this line